### ISSUE TITLE
Can't connect to Oracle - Invalid node version: 10.16.3

### ISSUE THREAD

**Issue by mtoma** [mtxr/vscode-sqltools#401]
**Describe the bug**
Connection to an Oracle DB fails with: "Failed to install dependencies for OracleDB: Request DependencyInstaller/install failed with message: Invalid node version: 10.16.3"

**To Reproduce**
Create an Oracle DB connection and click "Connect"

**Expected behavior**
Should connect to the DB

**Screenshots**
![Screenshot_20191002_120351](https://user-images.githubusercontent.com/5492426/66036096-bb85ff00-e50c-11e9-9146-4f2fff051ce2.png)

**Desktop (please complete the following information):**
 - SQLTools Version: 0.21.0
 - VSCode Version: 1.38.1
 - Dialect Used/Version: Oracle
 - OS: Linux (OpenSUSE Tumbleweed)

**Additional context**
node --version: v10.16.3
oracledb ** Node-oracledb 4.0.1 installed for Node.js 10.16.3 (linux, x64)

**Log:
```
INFO: Executing SQLTools.selectConnection
ERROR: Error:  { [Error: Request DependencyInstaller/install failed with message: Invalid node version: 10.16.3

	at /home/user/.vscode/extensions/mtxr.sqltools-0.21.0/extension.js:1:22985
	at /home/user/.vscode/extensions/mtxr.sqltools-0.21.0/extension.js:1:23392
	at Immediate.module.exports.messageQueue.size.setImmediate (/home/user/.vscode/extensions/mtxr.sqltools-0.21.0/extension.js:1:23936)
	at runCallback (timers.js:694:18)
	at tryOnImmediate (timers.js:665:5)
	at processImmediate (timers.js:647:5)] code: -32603, data: undefined }
```

**Comment by mtxr** [comment]
@mtoma did you get it working?

**Comment by mtxr** [comment]
Just a question, how did you get SQLTools to work use 4.0.1 while in the code it's fixed in 4.0.0? Did you manually updated or installed the driver in the extension folder?

**Comment by mtxr** [comment]
Please, reopen if will still have issues after clearing deps cache reinstalling
